test/test_distributions.py

```diff
@@ -652,6 +665,29 @@ def test_pathwise_gradient(jax_dist, sp_dist, params):
         assert_allclose(actual_grad[i], expected_grad, rtol=0.005)
 
 
+@pytest.mark.parametrize(
+    "jax_dist, sp_dist, params", CONTINUOUS + DISCRETE + DIRECTIONAL
+)
+def test_jit_log_likelihood(jax_dist, sp_dist, params):
```

Comment:
Whereas jitting in `test_log_prob` fixes params and varies the `value`, jitting in this test fixes the `value` and varies params, as would be the case in an observe statement `numpyro.sample(..., obs=data)`.